src/test/java/tr/com/orioninc/laborant/repository/RepositoryTest.java

```diff
@@ -0,0 +1,75 @@
+package tr.com.orioninc.laborant.repository;
+
+import lombok.extern.log4j.Log4j2;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import tr.com.orioninc.laborant.model.Lab;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJpaTest
+@Log4j2
+public class RepositoryTest {
+
+    @Autowired
+    private LabRepository underTest;
+
+    @Test
+    void itShouldCheckIfLabExists() {
+        // given
+        String labName = "testLab";
+        Lab lab = new Lab(
+                labName, "testUser", "testPassword", "testHost", 22);
+        log.info(underTest.save(lab));
+        // when
+        Lab foundLab = underTest.findByLabName(labName);
+
+        // then
+        assertThat(foundLab).isNotNull();
```

Comment:
You can also check object fields like:
assertThat(foundLab).isNotNull()
.returns("testUser, Lab::getUser)
.returns(.....

Please also assert that test values are correct.